d2l-table-circle-button.html

```diff
@@ -0,0 +1,71 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../d2l-icons/d2l-icon.html">
+
+<!--
+# D2L Table Circle Button
+-->
+
+<dom-module id="d2l-table-circle-button">
+	<template strip-whitespace>
+		<style>
+			:host {
+				display: inline-block;
+				position: absolute;
+			}
+			button.d2l-table-circle-button::-moz-focus-inner {
+				border: 0;
+			}
+			button.d2l-table-circle-button {
+				background-color: var(--d2l-scroll-wrapper-background-color);
+				border: 1px solid var(--d2l-scroll-wrapper-border-color);
+				border-radius: 50%;
+				box-sizing: content-box;
+				cursor: pointer;
+				display: inline-block;
+				font-family: inherit;
+				font-size: inherit;
+				height: 18px;
+				line-height: 0;
+				margin: 0;
+				padding: 10px;
+				text-decoration: none;
+				width: 18px;
+			}
+			button.d2l-table-circle-button:hover,
+			button.d2l-table-circle-button:focus {
+				outline-style: none;
+			}
+			button.d2l-table-circle-button > d2l-icon {
+				transition: color 0.3s ease;
+			}
+			button.d2l-table-circle-button:hover > d2l-icon,
+			button.d2l-table-circle-button:focus > d2l-icon {
+				color: var(--d2l-color-celestine-minus-1);
```

Comment:
Explicitly import `d2l-colors.html` when you reference one of its variables. This will work because other things are importing it, but if they stopped for whatever reason, the variable wouldn't be defined.